client/package.json

```diff
@@ -371,6 +371,7 @@
   },
   "prettier": {
     "arrowParens": "avoid",
-    "semi": false
+    "semi": false,
+    "trailingComma": "none"
```

Comment:
I think this is such a small change that doesn't need a brand new pull request. The reason why this exists is that when we run `yarn run lint-fix`, prettier formats with trailing commas and eslint removes those commas. Making build system fail for a split second, and cause re build + ide and git warn about the problems in the code for nothing. If we don't already allow trailing commas, we should also prevent prettier to format that way, even for a split second.

Reply:
It should at least be in a separate commit, with this explanation in there. Otherwise, let's say this breaks something else ... and therefore we will look at this later, we would have no idea why this change was made.